server/model/monitor.js

```diff
@@ -359,6 +359,9 @@ class Monitor extends BeanModel {
                             } catch (e) {
                                 throw new Error("Your JSON body is invalid. " + e.message);
                             }
+                        } else if (this.httpBodyEncoding === "form") {
+                            bodyValue = this.body;
+                            contentType = "application/x-www-form-urlencoded; charset=utf-8";
```

Comment:
See https://stackoverflow.com/questions/16819502/application-x-www-form-urlencoded-and-charset-utf-8
=> not needed

```suggestion
                            contentType = "application/x-www-form-urlencoded";
```